packages/client/hmi-client/src/components/asset/tera-asset.vue

```diff
@@ -14,7 +14,11 @@
 			</aside>
 		</header>
 		<template v-if="!hideIntro">
-			<header id="asset-top" ref="headerRef">
+			<header
+				id="asset-top"
+				:class="pageType === ProjectPages.OVERVIEW && 'overview-banner'"
```

Comment:
Use [class binding](https://vuejs.org/guide/essentials/class-and-style.html#binding-html-classes) instead.
```suggestion
				:class="{ 'overview-banner': pageType === ProjectPages.OVERVIEW }"
```